src/daemons/StorageDaemon.cpp

```diff
@@ -50,7 +50,7 @@ static void signalHandler(int sig);
 static Status setupSignalHandler();
 
 
-std::unique_ptr<nebula::kvstore::KVStore> getStoreInstance(
+std::shared_ptr<nebula::kvstore::KVStore> getStoreInstance(
```

Comment:
Why shared_ptr?

Reply:
seems can't copy unique_ptr

Reply:
Where did you need to copy it ?  If in httpHandler,  you could just pass the raw pointer inside.

Reply:
why copy. looks like a singleton.  do you need multi-instance?